src/api/app/models/project.rb

```diff
@@ -89,9 +89,7 @@ def autocomplete(search)
     where.not('name rlike ?', ::Configuration.unlisted_projects_filter) if ::Configuration.unlisted_projects_filter.present?
   }
   scope :remote, -> { where('NOT ISNULL(projects.remoteurl)') }
-  scope :autocomplete, lambda { |search|
-    where('lower(name) like lower(?)', "#{search}%").where.not('lower(name) like lower(?)', "#{search}%:%")
-  }
+  scope :autocomplete, ->(search) { where('lower(name) like lower(?)', "#{search}%") }
```

Comment:
Removing the not search for `home:`?!?!? But it is great! :angry:

Reply:
Could you also provide an argument? And it's actually a not search in sub-projects. So if I want to branch `OBS:Server:Unstable:Foo` I would have to enter `OBS:Server:Unstable` until the autocomplete offers me `OBS:Server:Unstable:Foo`